[TextInput] Update variants (#2327)
KissKissBankBank/kitten: CHANGELOG.md

## [unreleased]

- Fix: `VerticalStepper` mobile style fixes.

> - Feature: Add `regular` and `big` props on the `andromeda` and `orion` variants of `TextInput` and `regular` prop becomes the default prop.

Reviewer: Je propose plus court :

```suggestion
- Feature: Add `regular` (default) and `big` props on all variants of `TextInput`.
```
